### PR TITLE
[Snapshot] Store the current index version on MinIO | Update the local siren-reader alias after each index

### PR DESCRIPTION
**Pull request by MKCG**
This PR introduces 3 changes :

1. The `siren-reader` alias is also locally updated after a successful indexing for compatibility with the dev instance
2. The new daily index version is stored on MinIO
3. A new DAG to restore the previous index version based on the previous index version stored on MinIO

### CODE REVIEW

File: workflows/data_pipelines/elasticsearch/task_functions/snapshot.py [hunk 1]
```diff
@@ -10,7 +10,77 @@
     ELASTIC_PASSWORD,
     ELASTIC_SNAPSHOT_REPOSITORY,
     ELASTIC_SNAPSHOT_MAX_REVISIONS,
+    ELASTIC_SNAPSHOT_MINIO_STATE_PATH,
 )
+from dag_datalake_sirene.helpers.minio_helpers import minio_client
+
+
+def update_minio_current_index_version(**kwargs):
```
Comment (HAEKADI) [inline workflows/data_pipelines/elasticsearch/task_functions/snapshot.py:18]: Add one liner for function purpose here if possible :)

File: helpers/minio_helpers.py [hunk 2]
```diff
@@ -199,5 +201,33 @@ def get_files_and_last_modified(self, prefix: str):
         logging.info(f"*****List of files: {file_info_list}")
         return file_info_list
 
+    def read_json_file(self, dest_path, dest_name):
```
Comment (HAEKADI) [inline helpers/minio_helpers.py:204]: If possible to use existing methods within the class and add helpers for temp file creation. that would be great !

**Review by HAEKADI — CHANGES_REQUESTED** [review]

**Comment by MKCG** [comment]
Relates to https://github.com/etalab/annuaire-entreprises-search-infra/pull/258

**Review by HAEKADI — APPROVED** [review]
🚀

**Pull request merged**